app.js

```diff
@@ -20,22 +20,23 @@ const serveFileToHtml = require('./src/fileServer.js');
 const initialize = require('./src/initializer.js');
 const db = require('./src/db-conn.js');
 const dbInteract = require('./src/auth.js');
+const constants = require('./constants/index');
 
 const app = express();
-const PORT = argv.port || 8080;
-const ADDRESS = argv.address || '127.0.0.1';
-const connURL = argv.mongo || "mongodb://127.0.0.1:27017/keys";
+const PORT = argv.port || constants.PORT;
+const ADDRESS = argv.address || constants.ADDRESS;
+const connURL = argv.mongo || constants.URI;
```

Comment:
It might go against the notion of 'constants' but I intended that this file would also handle the argument parsing, which is still done in app.js. And thus, if another file would be accessing the port number, it will not be correct.

Reply:
Note that the `./src/key-creator.js` uses only the localhost connection, but this has a choice, therefore you will need two variables there.

Reply:
Sure I will change this

Reply:
So in key-creator.js need to also add the mongo uri string coming as argument?

Reply:
Yes.
Something like this
```
MONGO_URI_LOCAL = "..localhost.."
MONGO_URI = argv.mongo || MONGO_URI_LOCAL
```